src/components/Tokens/TokenTable/TokenRow/index.tsx

```diff
@@ -0,0 +1,254 @@
+  fill: ${({ isFavorited, theme }) => (isFavorited ? theme.accentAction : 'transparent')};
+`
+const TokenName = styled.div`
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  max-width: 100%;
+`
+export const LogoContainer = styled.div`
+  position: relative;
+  align-items: center;
+  display: flex;
+`
+export const L2NetworkLogo = styled.div<{ networkUrl?: string; size?: string }>`
+  height: ${({ size }) => size ?? '12px'};
+  width: ${({ size }) => size ?? '12px'};
+  position: absolute;
+  left: 50%;
+  bottom: 0;
+  background: url(${({ networkUrl }) => networkUrl});
+  background-repeat: no-repeat;
+  background-size: ${({ size }) => (size ? `${size} ${size}` : '12px 12px')};
+  display: ${({ networkUrl }) => !networkUrl && 'none'};
+`
+
+/* Loading State: row component with loading bubbles */
+export function TokenRowCells({
+  index,
+  name,
+  price,
+  percentChange,
+  tvl,
+  volume,
+  sparkline,
+}: {
+  loading?: boolean
+  index: ReactNode
+  name: ReactNode
+  price: ReactNode
+  percentChange: ReactNode
+  tvl: ReactNode
+  volume: ReactNode
+  sparkline?: ReactNode
+}) {
+  return (
+    <>
+      <ListNumberCell>{index}</ListNumberCell>
+      <NameCell>{name}</NameCell>
+      <PriceCell>
+        <PriceInfoCell>
+          {price}
+          <PercentChangeInfoCell>{percentChange}</PercentChangeInfoCell>
+        </PriceInfoCell>
+      </PriceCell>
+      <PercentChangeCell>{percentChange}</PercentChangeCell>
+      <TvlCell>{tvl}</TvlCell>
+      <VolumeCell>{volume}</VolumeCell>
+      <SparkLineCell>{sparkline}</SparkLineCell>
+    </>
+  )
+}
+
+interface TokenRowProps {
+  token: NonNullable<TopToken>
+  tokenListRank: number
+  l2CircleLogo: string | undefined
+  sparklineMap: SparklineMap
+}
+
+export default function TokenRow({ token, l2CircleLogo, tokenListRank, sparklineMap }: TokenRowProps) {
+  const delta = token.market?.pricePercentChange?.value
+  const arrow = getDeltaArrow(delta)
+  const formattedDelta = formatDelta(delta)
+
+  return (
+    <StyledTokenRow>
+      <TokenRowCells
+        index={tokenListRank}
+        name={
+          <>
+            <LogoContainer>
+              <CurrencyLogo src={useTokenLogoURI(token)} symbol={token.symbol} />
+              <L2NetworkLogo networkUrl={l2CircleLogo} />
+            </LogoContainer>
+            <TokenInfoCell>
+              <TokenName>{token.name}</TokenName>
+              <TokenSymbol>{token.symbol}</TokenSymbol>
+            </TokenInfoCell>
+          </>
+        }
+        price={formatDollar({ num: token.market?.price?.value, isPrice: true, lessPreciseStablecoinValues: true })}
+        percentChange={
+          <>
+            {formattedDelta}
+            {arrow}
+          </>
+        }
+        tvl={formatDollar({ num: token.market?.totalValueLocked?.value })}
+        volume={formatDollar({ num: token.market?.volume?.value })}
```

Comment:
Is this no longer clickable? Or has that been moved to TokenRowCells? Same with above.